Sources/Core/AWSClientRuntime/Auth/AuthSchemes/SigV4AuthScheme.swift

```diff
@@ -12,10 +12,38 @@ public struct SigV4AuthScheme: ClientRuntime.AuthScheme {
     public let signer: ClientRuntime.Signer = AWSSigV4Signer()
     public let idKind: ClientRuntime.IdentityKind = .aws
 
+    public init() {}
+
     public func customizeSigningProperties(signingProperties: Attributes, context: HttpContext) -> Attributes {
         var copy = signingProperties
```

Comment:
nit: copy is a bit unclear maybe `updatedSigningProperties` or `signingPropertiesWithAdditionalAttributes`

Reply:
Changed to `updatedSigningProperties` as suggested.